tests/test_sealed_box.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2013 Donald Stufft and individual contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import, division, print_function
+
+import binascii
+
+import pytest
+
+from nacl.encoding import HexEncoder
+from nacl.public import PrivateKey, PublicKey, SealedBox
+
+
+VECTORS = [
+    # privalice, pubalice, plaintext, encrypted
+    (
+        b"77076d0a7318a57d3c16c17251b26645df4c2f87ebc0992ab177fba51db92c2a",
```

Comment:
Access to work environment needed... Will answer in a couple days. Thank you very much for the review.